### ISSUE TITLE
Inconsistent versioning

### ISSUE THREAD

**Issue by iNikem** [nebula-plugins/nebula-release-plugin#187]
I have a project with latest tag `v0.2.2`. I have configured it to use SNAPSHOT version strategy as default:
```
release {
  defaultVersionStrategy = nebula.plugin.release.git.opinion.Strategies.SNAPSHOT
}
```

Now, `./gradlew build` results in `Inferred project: trace-java, version: 0.2.3-SNAPSHOT`.
But `./gradlew snapshot` results in `Inferred project: trace-java, version: 0.3.0-SNAPSHOT`.
Which does not make sense to me. In both cases I want to build and publish the same snapshot version, no?

**Comment by kzhu2012** [comment]
Try ./gradlew snapshot -Prelease.scope=patch, I think you will get version: 0.2.3-SNAPSHOT as well.

snapshot is a task by nebular.release plugin, with a default release scope of minor.
build is a task by gradle base plugin, guess the default release scope is patch?

**Comment by iNikem** [comment]
Yes, it is indeed seems like different scopes. I don't have any specific proposal, what to do with this, but current behaviour does seem confusing.

**Comment by kzhu2012** [comment]
iNikem,
I am fairly new to nebular.release plugin, and gradle release/publish task.  One thing I found out for nebular.release is that it's very restrictive for release tag format, only **v1.2.3** format is supported, and all other format will not be recognized at all.   We are trying to adopt the YYYY.MM.DD.xx format, like **v2020.05.14.1** (date plus release number), but it's not permitted.   
Any experience with the tag format?

**Comment by iNikem** [comment]
Nope. I am quite comfortable with standard semver format.

**Comment by rpalcolea** [comment]
Hi folks! 

Sorry for taking so long

This should be fixed in 15.0.1 released last night (California time 😅 )
